x-pack/test/functional/apps/maps/embeddable/save_and_return.js

```diff
@@ -99,7 +99,7 @@ export default function ({ getPageObjects, getService }) {
       describe('save as', () => {
         it('should return to dashboard and add new panel', async () => {
           await PageObjects.maps.saveMap('Clone of map embeddable example');
-          await PageObjects.dashboard.waitForRenderComplete();
+          await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
This test is supposed to cover the use case that maps Save As adds another panel to the dashboard.

Occasionally, these panels fail to fully render. This happens only in this scenario:
- The first load of a map embeddable
- When the screen size is small
- using the geoshapes index pattern in this test.

I am completely unable to reproduce this in normal usage and https://github.com/elastic/kibana/issues/100280 has been opened to track the root cause of this issue, and any potential fixes.

This change works around the flakiness of this test by waiting for page load instead of waiting for all panels to render.